sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Azure.ResourceManager.AppConfiguration.csproj

```diff
@@ -8,5 +8,6 @@
     </Description>
     <PackageTags>azure;management;appconfiguration</PackageTags>
     <NoWarn>$(NoWarn);AZC0001;AZC0008;</NoWarn>
+    <TemporaryUsePreviousGeneratorVersion>true</TemporaryUsePreviousGeneratorVersion>
```

Comment:
There is `IsMgmtClientLibrary` property that you can use to set this flag in the `Directory.Build.Data.props` file instead of modifying each of the csproj here.

Reply:
I want to be able to undo this flag package by package instead all at once.